x-pack/plugins/endpoint/public/applications/endpoint/components/header_nav.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { MouseEvent } from 'react';
+import { i18n } from '@kbn/i18n';
+import styled from 'styled-components';
+import { EuiTabs, EuiTab } from '@elastic/eui';
+import { useHistory, useLocation } from 'react-router-dom';
+
+export interface NavTabs {
+  name: string;
+  id: string;
+  href: string;
+}
+
+export const navTabs: NavTabs[] = [
+  {
+    id: 'home',
+    name: i18n.translate('xpack.endpoint.headerNav.home', {
+      defaultMessage: 'Home',
+    }),
+    href: '/',
+  },
+  {
+    id: 'management',
+    name: i18n.translate('xpack.endpoint.headerNav.management', {
+      defaultMessage: 'Management',
+    }),
+    href: '/management',
```

Comment:
@paul-tavares does the `basePath` concern apply here as well?

Reply:
@bkimmel no - here its ok because we're using this value against react-router which we initialize with a `basePath` (the one provided during plugin initialization). So these should be (absolute) relative to `basePath`